packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -909,10 +921,13 @@ export class MgtPersonCard extends MgtTemplatedComponent {
           @input=${(e: Event) => {
             this._chatInput = (e.target as HTMLInputElement).value;
             this.requestUpdate();
-          }}>
+          }}
+          @keydown="${(e: KeyboardEvent) => this.sendQuickMessageOnEnter(e)}">
         </fluent-text-field>
-        <span class="send-message-icon" @click=${() => this.sendQuickMessage()}>
-          ${getSvg(SvgIcon.Send)}
+        <span class="send-message-icon" 
```

Comment:
Looks like a button, so needs role, tabindex and keypress to behave correctly, or just use a button